### PR TITLE
Bug 1215476 - [Test] task_manager_icon_test Marionette test failing l…

### PR DESCRIPTION
**Pull request by mcav**
…ocally.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f94aef516f3cfbe6c8d7a39862aa4bb715b82398

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0e3a1f32e761e34633566dcd38a45c20f6981d23

File: apps/system/test/marionette/task_manager_icons_test.js [hunk 1]
```diff
@@ -94,13 +101,16 @@ marionette('Task Manager - Icons', function() {
                icon.getAttribute('class').indexOf('pending') === -1;
       });
 
-      var expectedColor = '0,255,255,255';
-      var expectedSize = 32;
+      // The system will select an icon based on the current devicePixelRatio.
+      var expectedSize = 32 * client.executeScript(function() {
```
Comment (albertopq) [inline apps/system/test/marionette/task_manager_icons_test.js]: Shouldn't the expected size be '16 || 32 || 64 || 128 || 256'? If I understand this correctly, that code won't work. We should find the closest size to 32 * devicePixelRatio.

Comment (albertopq) [inline apps/system/test/marionette/task_manager_icons_test.js]: Well, it will work if devicePixelRatio == 2, but it won't on 1.5 or 2.2

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7844276bbac9ac76ebe7b228697b7fe9d48975c1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1c0b78c69106c97ee3e734c626a26b3764dd1330

**Pull request merged**